pkg/controllers/core/deployer_controller.go

```diff
@@ -20,22 +20,25 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/go-logr/logr"
-	"github.com/google/go-cmp/cmp"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/api/networking/v1beta1"
```

Comment:
`v1beta1` isn't a meaningful package name

```suggestion
	networkingv1beta1 "k8s.io/api/networking/v1beta1"
```